estimate densities from point patterns

ThierryO opened the pull request:
@stijnvanhoey can you render this? I can't render the SQL tutorial.

ThierryO committed:
add rmd file on estimating densities from point patterns

florisvdh committed:
point-pattern: move files in correct location

florisvdh committed:
point-pattern: rename Rmd

florisvdh commented:
Some work to get this tutorial in the right directory structure has been done in the `density_fv` branch. Fixing CRS specification will folllow later.

florisvdh committed:
spatial_point_pattern: underscores in folder

florisvdh committed:
spatial_point_pattern: add gfm md_document outputformat in YAML

florisvdh committed:
spatial_point_pattern: use PROJ>=6 compliant CRS

florisvdh committed:
spatial_point_pattern: use proj4string<- function rather than the slot *

The use of 'proj4string' in R code is confusing in times where PROJ.4 strings are obsoleted.
It can be expected that a more general function name to set an object's CRS will sooner or later
be promoted instead of proj4string<-, hence preparing for that already by using the function instead of the slot.
I expect the proj4string slot will most probably remain in use for compatibility reasons, but
explicitly naming it is better avoided in times where WKT2 strings are actually used to define a CRS.

florisvdh committed:
spatial_point_pattern: expand on CRS/proj4string/WKT2

florisvdh committed:
spatial_point_pattern: improve markdown & language

florisvdh reviewed (commented):
@ThierryO In PR #195 I suggest some changes to get in line with the current tutorials (folder setup, CRS).

Please note:

- `index.md` is still to be regenerated - reason: see next point. YAML header in Rmd has been updated already in #195 .
- the `ggmap` code with google map will need to be revised. Current code cannot be run (hence, `index.md` cannot be updated) except when getting a personal API from google and handing your credit card details (and it's free just for a limited period). Not fit for a tutorial anymore IMHO. Maybe you will now rather prefer using OSM.

florisvdh commented:
> @ThierryO In PR #195 I suggest some changes to get in line with the current tutorials (folder setup, CRS).
> 
> Please note:
> 
>   * `index.md` is still to be regenerated - reason: see next point. YAML header in Rmd has been updated already in #195 .
> 
>   * the `ggmap` code with google map will need to be revised. Current code cannot be run (hence, `index.md` cannot be updated) except when getting a personal API from google and handing your credit card details (and it's free just for a limited period). Not fit for a tutorial anymore IMHO. Maybe you will now rather prefer using OSM.

Follow-up: PR #195 has now been merged into this PR. @ThierryO the above notes still need some work before it can be merged in `master` and this long-standing but nice tutorial will be merged.

ThierryO commented:
Sorry. This has very low priority to me. Feel free to update the code.

florisvdh commented:
@inbo/bmk

peterdesmet committed:
Use frontmatter used elsewhere

See #285

florisvdh committed:
Merge pull request #306 from inbo/pr_template

Update PR template

hansvancalster committed:
Apply suggestions from code review

Co-authored-by: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

hansvancalster committed:
relative weblinks

hansvancalster committed:
open source TOTP tools

hansvancalster committed:
drop sentence about HTTP

hansvancalster committed:
fix anchor links

hansvancalster committed:
remove faulty claim about GitHub's default

hansvancalster committed:
update md file

hansvancalster committed:
fix section cross-references

hansvancalster committed:
redundant chunk options

hansvancalster committed:
add backticks

hansvancalster committed:
remove vaccinate instructions

hansvancalster committed:
update md

hansvancalster committed:
check global config settings

hansvancalster committed:
project-specific gitignore file from checklist

hansvancalster committed:
update md

hansvancalster committed:
update title to mention GitHub

- 2FA = about GitHub authentication

hansvancalster committed:
clarifications about 2FA

hansvancalster committed:
move sentence to more logical place

hansvancalster committed:
foolproofing...;-)

ThierryO committed:
bugfix in RProfile.site

ThierryO committed:
Merge pull request #312 from inbo/hotfix

bugfix in RProfile.site

ThierryO committed:
use remotes::install_cran() instead of utils::install.packages()

ThierryO committed:
streamline installation

ThierryO committed:
automatically remove any remainig lock file due to failed prior installation

ThierryO committed:
update markdown file

ThierryO committed:
use default repositories

ThierryO committed:
make the user remove 00LOCK manually

ThierryO committed:
instruct user to run R instead of the installer

ThierryO committed:
update markdown

ThierryO committed:
display error when INBO package are missing

Don't try to install missing packages .

ThierryO committed:
render index.md

florisvdh committed:
Merge pull request #314 from inbo/hotfix2

Hotfix RProfile.site

hansvancalster committed:
Update content/tutorials/git_authentication/index.Rmd

Co-authored-by: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

hansvancalster committed:
address review comments Floris

ThierryO committed:
missing required INBO packages yields a warning instead of an error.
It is only displayed in interactive sessions.

ThierryO committed:
Merge pull request #315 from inbo/hotfix

missing required INBO packages yields a warning instead of an error.

hansvancalster committed:
minor tweaks

hansvancalster committed:
update md file

hansvancalster committed:
Update content/tutorials/git_authentication/index.Rmd

Co-authored-by: Els Lommelen <els.lommelen@inbo.be>

hansvancalster committed:
Update content/tutorials/git_authentication/index.Rmd

Co-authored-by: Els Lommelen <els.lommelen@inbo.be>

hansvancalster committed:
Update content/tutorials/git_authentication/index.Rmd

Co-authored-by: Els Lommelen <els.lommelen@inbo.be>

hansvancalster committed:
address review comments

hansvancalster committed:
remove accidental carriage return

hansvancalster committed:
better explain for whom tutorial is intended

hansvancalster committed:
add weblinks to install TOTP mobile apps

hansvancalster committed:
update md file

hansvancalster committed:
Merge pull request #309 from inbo/issue-287-git-authentication

closes #287. Thanks @ElsLommelen and @florisvdh for reviewing!

ThierryO committed:
render index.md

ThierryO committed:
Merge pull request #316 from inbo/hotfix

render index.md

damianooldoni committed:
Remove tutorial gbif issue

Vignette removed in https://github.com/ropensci/rgbif/commit/5bea0027b188eddccc559069ec2f2bd81d4cdd59

damianooldoni committed:
Split tutorial in two

Vignette has been removed and replaced by two vignettes: one for occurrences, one for species lists

damianooldoni committed:
Add new vignettes

damianooldoni committed:
Update vignette taxon names

damianooldoni committed:
update authors rgbif vignette

sckott removed because he is not present in any tutorial anymore.

damianooldoni committed:
Improve author filename

damianooldoni committed:
Use id as author's filename

damianooldoni committed:
Update version of checkout and upload-artifact actions

@florisvdh, @ThierryO: as mentioned in warning:
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: actions/checkout@v2, actions/upload-artifact@v2

damianooldoni committed:
Merge pull request #318 from inbo/improve-rgbif-vignettes

Improve rgbif tutorials linked to rgbif vignettes

damianooldoni committed:
Add Rmd and images

damianooldoni committed:
Finalize Rmd and render md

damianooldoni committed:
Add R part about datasets at INBO

damianooldoni committed:
Add section about INBO as hosting institute

damianooldoni committed:
Add link to rgbif in intro

damianooldoni committed:
Rephrase intro

damianooldoni committed:
Rephrase sentence reason behind tutorial

Co-authored-by: Hans Van Calster <hans.vancalster@inbo.be>

damianooldoni committed:
Improve punctuation

Co-authored-by: Hans Van Calster <hans.vancalster@inbo.be>

damianooldoni committed:
Add forgotten text: datasetKey

Co-authored-by: Hans Van Calster <hans.vancalster@inbo.be>

damianooldoni committed:
Improve sentence

Co-authored-by: Hans Van Calster <hans.vancalster@inbo.be>

damianooldoni committed:
Add article

Co-authored-by: Hans Van Calster <hans.vancalster@inbo.be>

damianooldoni committed:
Avoid repetition

Co-authored-by: Hans Van Calster <hans.vancalster@inbo.be>

damianooldoni committed:
Add "also"

Co-authored-by: Hans Van Calster <hans.vancalster@inbo.be>

damianooldoni committed:
Knit to md

Final knitting to render the Rmd as md. Thanks, @hansvancalster, for the reminder!

damianooldoni committed:
Merge pull request #319 from inbo/retrieve_gbif_checklist_in_r

Add tutorial about how to get GBIF checklist datasets via `rgbif`

florisvdh committed:
r_gbif_checklist: some fixes

florisvdh committed:
r_gbif_checklist: another text fix (Rmd only)

damianooldoni committed:
Improve title and description

Co-Authored-By: Peter Desmet <peter.desmet@inbo.be>

damianooldoni committed:
Improve parts about keys

Co-Authored-By: Peter Desmet <peter.desmet@inbo.be>

damianooldoni committed:
Knit

damianooldoni committed:
Update content/tutorials/r_gbif_checklist/index.Rmd

Co-authored-by: Peter Desmet <peter.desmet@inbo.be>

damianooldoni committed:
rephrase

Co-authored-by: Peter Desmet <peter.desmet@inbo.be>

damianooldoni committed:
rephrase taxonID

Co-authored-by: Peter Desmet <peter.desmet@inbo.be>

damianooldoni committed:
Improve nubKey description

Co-authored-by: Peter Desmet <peter.desmet@inbo.be>

damianooldoni committed:
Correct typo

damianooldoni committed:
Knit again

damianooldoni committed:
Merge pull request #320 from inbo/r_gbif_checklist_fixes

r_gbif_checklist: some fixes

florisvdh committed:
git_zenodo: new tutorial

florisvdh committed:
git_zenodo: update title

florisvdh committed:
git_zenodo: extend wrt communities

florisvdh committed:
git_zenodo: add notes wrt editing metadata

florisvdh committed:
git_zenodo: elaborate on good git tags

Co-authored-by: Thierry Onkelinx <ThierryO@users.noreply.github.com>

florisvdh committed:
git_zenodo: add alternative to find URLs (thanks @ThierryO)

florisvdh committed:
git_zenodo: commit new images that I forgot to stage

florisvdh committed:
git_zenodo: refer to checklist for .zenodo.json automation (thanks @ThierryO)

florisvdh committed:
git_zenodo: use packages.inbo.be instead of inbo.github.io

florisvdh committed:
Merge pull request #321 from inbo/zenodo

Tutorial: Zenodo - GitHub integration

florisvdh committed:
spatial_mgrs: start article on MGRS

florisvdh committed:
Allow raw HTML to be left as-is by Hugo *

It is observed that the screenshot is no longer inserted in index.md as markdown figure,
but as a HTML figure. I did not find how to control this behaviour.

Using rmarkdown + webshot/webshot2 + pandoc 3.1.3 + gfm output (unsure where the cause lies).

By default Hugo ignores raw HTML output, but this can be worked around.

Source: https://stackoverflow.com/questions/63198652/hugo-shortcode-ignored-saying-raw-html-omitted#63206852

florisvdh committed:
spatial_wms: update WMS URL

florisvdh committed:
Merge pull request #324 from inbo/wms

spatial_wms: update WMS URL

florisvdh committed:
spatial_mgrs: various minor improvements

florisvdh committed:
spatial_mgrs: improve section on software

florisvdh committed:
spatial_mgrs: major rewrite & expansion

florisvdh committed:
spatial_mgrs: knit

hansvancalster committed:
update url's and mention inbospatial

hansvancalster committed:
Merge pull request #326 from inbo/fix-issue-#325

update WFS tutorial

florisvdh committed:
Merge branch 'master' into mgrs

florisvdh committed:
spatial_mgrs: link citations

florisvdh committed:
spatial_mgrs: textual updates

florisvdh committed:
spatial_mgrs: knit

florisvdh committed:
spatial_mgrs: add figures

florisvdh committed:
spatial_mgrs: improve portions of UTM referencing

florisvdh committed:
spatial_mgrs: improve figures & captions

florisvdh committed:
spatial_mgrs: language fix

florisvdh committed:
spatial_mgrs: update date

florisvdh committed:
spatial_mgrs: fix csl filepath (no tempdir since index.md hardcodes it)

florisvdh committed:
CSS: use italic, smaller font & padding for captions

florisvdh committed:
spatial_mgrs: knit and include all assets

florisvdh committed:
spatial_mgrs: include support for pdf output

florisvdh committed:
Revert "spatial_mgrs: fix csl filepath" and drop the csl file *

This reverts commit a5f67dd11fc02ddbb31f39a585529da5ad899a5e and drops the CSL file again
(added in b78e67a).

I had tricked myself into believing that the YAML of index.md is being used (and it
contains the temp path to the csl file). However it isn't and index.md is already the
result of applying the CSL style, so no CSL file lookup needed here.

florisvdh committed:
spatial_mgrs: fix spelling & grammar

Co-authored-by: Ward Langeraert <60934857+wlangera@users.noreply.github.com>

florisvdh committed:
spatial_mgrs: update coordinate output formatting

Co-authored-by: Ward Langeraert <60934857+wlangera@users.noreply.github.com>

florisvdh committed:
spatial_mgrs: flip args in coord formatting code for educational reasons

florisvdh committed:
spatial_mgrs: reword sentence in MGRS criticisms (thanks @wlangera)

florisvdh committed:
spatial_mgrs: improve punctuation

florisvdh committed:
spatial_mgrs: adopt easting/northing terms (thanks @hansvancalster)

florisvdh committed:
spatial_mgrs: explain main lettering principles of MGRS

florisvdh committed:
spatial_mgrs: knit

florisvdh committed:
spatial_mgrs: code style fix

florisvdh committed:
spatial_mgrs: add NGA's own advice on when (not) to use MGRS

florisvdh committed:
spatial_mgrs: improve figure caption

florisvdh committed:
spatial_mgrs: refer to NGA's appendix wrt making UTM grid references unique

florisvdh committed:
spatial_mgrs: minor addition to MGRS lettering principles

florisvdh committed:
spatial_mgrs: use same location as UTM-grid in MGRS example

florisvdh committed:
spatial_mgrs: improve wording & grammar

florisvdh committed:
spatial_mgrs: move part of summary to intro

florisvdh committed:
spatial_mgrs: add grid reference examples in summary (thanks @ToonHub)

florisvdh committed:
spatial_mgrs: explain the grid pattern complexities of UTM & MGRS better

florisvdh committed:
spatial_mgrs: further explain degree of match between UTM & MGRS

florisvdh committed:
spatial_mgrs: minor layout update

florisvdh committed:
spatial_mgrs: fix and better explain separators in UTM & MGRS

florisvdh committed:
spatial_mgrs: also refer to Snyder (1987)

florisvdh committed:
spatial_mgrs: minor rewordings

florisvdh committed:
spatial_mgrs: knit

florisvdh committed:
spatial_mgrs: MGRS is also in polar regions (summary)

florisvdh committed:
spatial_mgrs: improve wording in summary

florisvdh committed:
spatial_mgrs: better explain historical entwinement between UTM & MGRS

florisvdh committed:
spatial_mgrs: update date

florisvdh committed:
spatial_mgrs: knit

florisvdh committed:
spatial_mgrs: always use latest CSL version

florisvdh committed:
spatial_mgrs: address several comments from @jeverbeu and @robsonnasc (email)

florisvdh committed:
spatial_mgrs: add info & example about MGRS digits (thanks @jeverbeu, @robsonnasc)

florisvdh committed:
spatial_mgrs: update color in maps

florisvdh committed:
spatial_mgrs: align examples

florisvdh committed:
spatial_mgrs: minor fix

florisvdh committed:
spatial_mgrs: add acknowledgements

florisvdh committed:
spatial_mgrs: update date

florisvdh committed:
spatial_mgrs: knit

florisvdh committed:
custom.css: update style of class 'csl-entry' *

This adds space between bibliographic references.

Current Pandoc versions generate a HTML div element (class csl-entry) for each bib reference in its markdown
output format, thereby dropping whitelines between them. Before, simple markdown text was used.

florisvdh committed:
Merge pull request #327 from inbo/mgrs

A post about the MGRS and UTM grids

florisvdh committed:
GHA workflows: use latest Ubuntu LTS

florisvdh committed:
Merge pull request #328 from inbo/gha

GHA workflows: use latest Ubuntu LTS

hansvancalster committed:
add missing packages

hansvancalster committed:
add qgisprocess

hansvancalster committed:
Update content/articles/inbo_software/index.md

Co-authored-by: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

hansvancalster committed:
Merge pull request #329 from inbo/update-inbo-software-page

add missing packages

hansvancalster committed:
add geoprocessing section

florisvdh committed:
git_authentication: replace andOTP by Aegis *

andOTP is no longer maintained since 2022:
https://xdaforums.com/t/unmaintained-app-4-4-open-source-andotp-open-source-two-factor-authentication-for-android.3636993/page-6#post-87021655

In the same forum thread, Aegis is recommended as open-source alternative. It is indeed actively maintained
and I can testify that it works very well. Also, it can import the keys from andOTP.

See https://github.com/beemdevelopment/Aegis

florisvdh committed:
Merge pull request #330 from inbo/aegis

git_authentication: replace andOTP by Aegis

florisvdh committed:
Rprofile.site: add option qgisprocess.detect_newer_qgis = TRUE *

For upcoming versions of {qgisprocess}, upon loading the package it will detect the availability of a more recent standalone QGIS installation in Windows & macOS than the version currently set in {qgisprocess}. {qgisprocess} will then offer to use the newer version instead.

ThierryO committed:
Merge pull request #331 from inbo/florisvdh-patch-1

Rprofile.site: add option qgisprocess.detect_newer_qgis = TRUE

RCinbo committed:
Create raisacarmen.toml

Add a toml for raisa carmen

raiisac committed:
add authors .toml

raiisac committed:
Add index.md for the r_brms toturial

raiisac committed:
Refer to static htmls

The md files did not allow to show ggplot figures or to choose whether hide/show R code

raiisac committed:
Add static htmls

raiisac committed:
Add links to course material

raiisac committed:
Add english course material

raiisac committed:
Add Dutcj course material

florisvdh committed:
user_install_r: fix typo

ThierryO committed:
Merge pull request #333 from inbo/florisvdh-patch-1

user_install_r: fix typo

damianooldoni committed:
Improbve broken links to course material

damianooldoni committed:
Add link to brms package homepage

damianooldoni committed:
Add link to R script for installation

damianooldoni committed:
Improve link to installation R script

RCinbo committed:
Merge pull request #332 from inbo/RCinbo-bayesianstatistics

Add the bayesian statistics tutorial. Thanks Damiano for the review and fixes!

ThierryO committed:
tutorial on solving merge conflict on GitHub.com

ThierryO committed:
move tutorial to its own folder

ThierryO committed:
Apply suggestions from code review

Co-authored-by: Floris Vanderhaeghe <floris.vanderhaeghe@inbo.be>

ThierryO committed:
Merge pull request #334 from inbo/merge_conflict

tutorial on solving merge conflict on GitHub.com

ThierryO committed:
update Rprofile.site

fredericpiesschaert committed:
Create index.md

fredericpiesschaert committed:
Create index.md

fredericpiesschaert committed:
Update index.md

fredericpiesschaert committed:
Update index.md

fredericpiesschaert committed:
Update index.md

ThierryO committed:
Merge pull request #335 from inbo/install

update Rprofile.site

damianooldoni committed:
Update baseurl

damianooldoni committed:
Merge pull request #339 from inbo/adapt-settings-post-url-change

Adapt tech settings after change deployment domain

damianooldoni committed:
Remove `baseurl`

As for the coding club, @peterdesmet and I think the baseurl is not needed anymore.

damianooldoni committed:
Merge pull request #340 from inbo/remove-baseurl

Remove `baseurl`

ElsLommelen committed:
Merge branch 'master' into fredericpiesschaert-patch-1

EmmaCartuyvels1 committed:
Update index.md

Link was expired

ElsLommelen committed:
add author elslommelen

ElsLommelen committed:
correct names

damianooldoni committed:
Merge pull request #337 from inbo/fredericpiesschaert-patch-1

update inbodb tutorial page with Taxonlijsten

ElsLommelen committed:
Update installation instructions RStudio

ThierryO committed:
Merge pull request #345 from inbo/Update-suggestion-for-installation-instructions

Update installation instructions RStudio

falkmielke committed:
brms_rstan: general updates

EmmaCartuyvels1 committed:
Merge pull request #344 from inbo/EmmaCartuyvels1-patch-1

Update index.md

falkmielke committed:
brms_rstan: poisson model, plain+hierarchical

falkmielke committed:
brms_rstan: windows testing, index, author

falkmielke committed:
brms-rstan: wording

Co-authored-by: Ward Langeraert <60934857+wlangera@users.noreply.github.com>

falkmielke committed:
brms-rstan: wording (nl)

Co-authored-by: Ward Langeraert <60934857+wlangera@users.noreply.github.com>

falkmielke committed:
brms-rstan: wording (nl)

Co-authored-by: Ward Langeraert <60934857+wlangera@users.noreply.github.com>

falkmielke committed:
brms-rstan: wording (nl)

Co-authored-by: Ward Langeraert <60934857+wlangera@users.noreply.github.com>

falkmielke committed:
brms-rstan: wording (nl)

Co-authored-by: Ward Langeraert <60934857+wlangera@users.noreply.github.com>

falkmielke committed:
brms-rstan: rstan moved as suggested.

falkmielke committed:
brms-rstan: generalizing frequentists; translation

falkmielke committed:
brms-rstan: generalizing frequentists (en)

falkmielke committed:
brms-rstan: mentioning cmdstanr backend

falkmielke committed:
brms-rstan: re-organize static files

falkmielke committed:
brms-rstan: knit files; image issues

PietrH committed:
Switch to using artifacts v4

https://github.blog/changelog/2024-04-16-deprecation-notice-v3-of-the-artifact-actions/

damianooldoni committed:
Merge pull request #350 from inbo/349-switch-to-artifacts-v4

Switch to using artifacts v4

florisvdh committed:
r_database_access: update argument of dbConnect()

florisvdh committed:
Merge pull request #351 from inbo/database_access

r_database_access: update argument of dbConnect()

falkmielke committed:
Merge pull request #346 from inbo/rstan

BRMS tutorial: add `rstan` example

ThierryO committed:
fix url

ThierryO committed:
Merge pull request #354 from inbo/inbomd

fix url

florisvdh committed:
Update pull_request_template.md

While reviewing https://github.com/inbo/tutorials/pull/356, the instructions of the PR template for reviewing seem to no longer work to render the site from the GHA artifact.

With these changes, it seems to work again though, using python -m http.server 8887

falkmielke committed:
update_pr_template: introduce

falkmielke committed:
update_pr_template: *cf.* consistency

Latijns -> italics; https://nl.wikipedia.org/wiki/Cf.

falkmielke committed:
update_pr_template: proofread "create tutorial"

falkmielke committed:
update_pr_template: refresh pr template

falkmielke committed:
update_pr_template: additional suggestions*

by @florisvdh, https://github.com/inbo/tutorials/pull/357

florisvdh committed:
Merge pull request #357 from inbo/update_pr_template

Update pull_request_template.md

falkmielke committed:
density: remove ancient _site.yml

falkmielke committed:
density: revival

falkmielke committed:
density: rebase / merge conflicts

falkmielke commented:
Sorry for necro-bumping...
I rebased this branch and added an updated little tutorial to calculate point density.

Feel free to add/comment/wrap up and merge @ThierryO
